easybuild/easyblocks/generic/binary.py

```diff
@@ -124,6 +124,10 @@ def post_install_step(self):
 
         super(Binary, self).post_install_step()
 
+    def sanity_check_rpath(self):
+        """Skip the rpath sanity check, this is binary software"""
+        pass
```

Comment:
@pescobar I would emit a clear log message rather than just skipping this method:

```python
self.log.info("Skipping RPATH sanity check for binary installations")
```